compiler/rustc_parse/src/parser/expr.rs

```diff
@@ -2225,7 +2225,7 @@ impl<'a> Parser<'a> {
                     attrs: attrs.into(),
                     ty,
                     pat,
-                    span: lo.to(this.token.span),
+                    span: lo.to(this.prev_token.span),
```

Comment:
This change means that we no longer include the comma in the param span, right? I don't know if that is a good call 🤔 

Maybe we can address the span issue on the other end, by creating a new span that is `param.pat.span.to(param.ty.map_or(param.pat.span, |ty| ty.span)`. I guess it depends on what we've used the param's span for elsewhere (even though there haven't been any visible changes in our test suite from this... I guess I'm fine either way :-/

Reply:
I think nobody used it, evidenced by the lack of UI test changes. 

This makes it more consistent with  regular fn signatures' spans.

Reply:
That's fine, let's go ahead with that change then. We can revisit this if we ever need the param + comma (I'm sure, for removal suggestions) :)